pkg/opensearch/opensearch.go

```diff
@@ -60,25 +61,25 @@ func (ds *OpenSearchDatasource) QueryData(ctx context.Context, req *backend.Quer
 		return nil, err
 	}
 
-	errRefID, err := handleServiceMapPrefetch(ctx, osClient, req)
-	if err != nil {
-		return wrapServiceMapPrefetchError(errRefID, err)
+	response := handleServiceMapPrefetch(ctx, osClient, req)
+	if response != nil {
+		return response, nil
 	}
 
 	query := newQueryRequest(osClient, req.Queries, req.PluginContext.DataSourceInstanceSettings)
-	response, err := wrapError(query.execute(ctx))
+	response, err = wrapError(query.execute(ctx))
```

Comment:
what is context of removing the `:=` to `=`? (I am a bit of a Go noob btw)

Reply:
also should we keep this version of `wrapError` around vs using the newer wrap for this error? or in this case are we confident that this is an error owned by us